apps/web/src/lib/components/UserPage.svelte

```diff
@@ -9,30 +9,31 @@
 	const user = $derived(userService.user);
 	const token = $derived(authService.token);
 	let userAvatarUrl = $state($user?.picture);
+	$effect(() => {
+		console.log(userAvatarUrl);
+	});
```

Comment:
Left over?

Also small tip, there's an `$inspect()` rune now that'll do this for you including a few niceties